absl/base/config.h

```diff
@@ -334,6 +334,8 @@
 #define ABSL_HAVE_ALARM 1
 #elif defined(_MSC_VER)
 // feature tests for Microsoft's library
+#elif defined(__MINGW32__)
+// feature tests for mingw
```

Comment:
Please explain in this comment why `ABSL_HAVE_ALARM` is not defined for `__MINGW32__`.  I realize you've copied the form of the preceding comments, but they're not very helpful and I'd like to do better.  The `__EMSCRIPTEN__` example below is better.

It looks to me like the right comment is:
```
// mingw32 doesn't provide alarm(2):
// https://osdn.net/projects/mingw/scm/git/mingw-org-wsl/blobs/5.2-trunk/mingwrt/include/unistd.h
// mingw-w64 provides a no-op implementation:
// https://sourceforge.net/p/mingw-w64/mingw-w64/ci/master/tree/mingw-w64-crt/misc/alarm.c
```